src/mmw/js/src/draw/views.js

```diff
@@ -72,9 +73,11 @@ var ToolbarView = Marionette.LayoutView.extend({
 });
 
 var SelectAreaView = Marionette.ItemView.extend({
+    $label: $('#boundary-label'),
```

Comment:
This element *must* be on the body directly to achieve proper offsets from the viewpoint origin, rather than a parent element origin, so it is not included in this view's template (unfortunately).